config/locales/eng.yml

```diff
@@ -100,7 +100,7 @@ eng:
         header: 'Welcome to the Sara Alert System!'
         dear: 'Dear'
         info1: 'You have been enrolled in the Sara Alert monitoring system. We ask that you provide daily reports of your status. Simply click the button below and follow the on-screen instructions.'
-        info2: 'You will receive a similar reminder daily until your monitoring period has ended. If you have any questions please reach out to the public health organization that helped enroll you.'
+        info2: 'You will receive a similar reminder daily until your monitoring period has ended. '
```

Comment:
is there a reason why there's a space here and in the rest of the files?

Reply:
There is a space because that is the first sentence and one of the two sentences that start with `If you have ...` will be added directly after it. 

There were no mockups just talking with Kate and the PO team. The second sentence, when contact info is provided, says `please use the contact information below`. Do you think we need more wording?

Reply:
that's what I figured in regards to the spacing, but the space wasn't there before, so I guess my question is more why did that need to change because that probably shouldn't be handled in the yml file?

Reply:
Updated layout and wording in https://github.com/SaraAlert/SaraAlert/pull/925/commits/665568faf48de3876dfddaf26d48c50fd36eafbe

Reply:
I broke the sentence in half and left the space with the first sentence instead of starting the second with a space. I can move it to the second if we would prefer that instead

Reply:
Nothing in the yml file has spaces so I would not have them in there and then finagle the html to do it instead

Reply:
Agreed; if possible, keeping trailing whitespace from these locale files would be preferred

Reply:
removed in https://github.com/SaraAlert/SaraAlert/pull/925/commits/2ce41203371a7d5edb33c783dc51505d97208037